contrib/gitian-build.sh

```diff
@@ -230,6 +230,8 @@ echo ${COMMIT}
 if [[ $setup = true ]]
 then
     sudo apt-get install ruby apache2 git apt-cacher-ng python-vm-builder qemu-kvm qemu-utils
+    git clone https://github.com/BTCGPU/gitian.sigs.git
+    git clone https://github.com/BTCGPU/bitcoingold-detached-sigs.git
```

Comment:
There is no need to clone the repos here and even worse this would conflict with passing the repo url/directory via command line.

Reply:
Wow.... You are so fast 👍

Reply:
Yes but it helps when we build it for the first time. It wouldn't clone without the -setup option.

Reply:
You're right, it's in the original code but it's also wrong there (like a few other things). The only repo which one has to manually clone to have setup run successfully is the BTCGPU repo. And btw, all repos are mentioned in the docs (https://github.com/BTCGPU/BTCGPU/blob/master/doc/release-process.md#first-time--new-builders). So one should never end up with missing repos.

Reply:
@wilsonmeier Hmm... Seems that the core's guide isn't friendly to the users. I think we need to clarify the guides at the BTCGPU.

Reply:
Yes, the docs are the right place to make this clear. The script should not fetch them without any possibility to overwrite this via command line. Therefore i ask you to remove them again. Thank you.

Reply:
" If you're using the automated script (found in contrib/gitian-build.sh), then at this point you should run it with the "--setup" command. Otherwise ignore this. "

I think leaving the -setup with those automatic clone processes will be the best. For now, I can't find any duplicated processes except the manual one.

Reply:
Ok, I will remove then :D